apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -1,34 +1,51 @@
 # frozen_string_literal: true
 
 class ProjectsController < ApplicationController
+  
+  # GET /projects/:id
   def show
     # files in current project
   end
 
+  # GET /projects
   def index
     @projects = Project.all
   end
 
+  # GET /projects/new
   def new
     @project = Project.new
   end
 
+  # GET /projects/:id/edit
   def edit
     @project = Project.find(params[:id])
   end
 
+  # PATCH /projects/:id
+  def update
+    @project = Project.find(params[:id])
+
+    if @project.update(project_params)
+      @project.write_manifeset
```

Comment:
`project#update` should encapsulate whatever it needs to do - namely writing the manifest.